public/app/features/alerting/unified/components/receivers/TemplateForm.tsx

```diff
@@ -121,74 +123,123 @@ export const TemplateForm: FC<Props> = ({ existing, alertManagerSourceName, conf
-          error={errors?.content?.message}
-          invalid={!!errors.content?.message}
-          required
-        >
-          <div className={styles.editWrapper}>
-            <AutoSizer>
-              {({ width, height }) => (
-                <TemplateEditor
-                  value={getValues('content')}
-                  width={width}
-                  height={height}
-                  onBlur={(value) => setValue('content', value)}
-                />
+        <TemplatingGuideline />
+        <div className={styles.contentContainer}>
+          <div>
+            <Field label="Content" error={errors?.content?.message} invalid={!!errors.content?.message} required>
+              <div className={styles.editWrapper}>
+                <AutoSizer>
+                  {({ width, height }) => (
+                    <TemplateEditor
+                      value={getValues('content')}
+                      width={width}
+                      height={height}
+                      onBlur={(value) => setValue('content', value)}
+                    />
+                  )}
+                </AutoSizer>
+              </div>
+            </Field>
+            <div className={styles.buttons}>
+              {loading && (
+                <Button disabled={true} icon="fa fa-spinner" variant="primary">
+                  Saving...
+                </Button>
               )}
-            </AutoSizer>
+              {!loading && (
+                <Button type="submit" variant="primary">
+                  Save template
+                </Button>
+              )}
+              <LinkButton
+                disabled={loading}
+                href={makeAMLink('alerting/notifications', alertManagerSourceName)}
+                variant="secondary"
+                type="button"
+                fill="outline"
+              >
+                Cancel
+              </LinkButton>
+            </div>
           </div>
-        </Field>
-        <div className={styles.buttons}>
-          {loading && (
-            <Button disabled={true} icon="fa fa-spinner" variant="primary">
-              Saving...
-            </Button>
-          )}
-          {!loading && (
-            <Button type="submit" variant="primary">
-              Save template
-            </Button>
-          )}
-          <LinkButton
-            disabled={loading}
-            href={makeAMLink('alerting/notifications', alertManagerSourceName)}
-            variant="secondary"
-            type="button"
-            fill="outline"
-          >
-            Cancel
-          </LinkButton>
+          <TemplateDataDocs />
         </div>
       </FieldSet>
     </form>
   );
 };
 
+function TemplatingGuideline() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Alert title="Templating guideline" severity="info">
+      <div>
+        Grafana uses the{' '}
+        <a
+          href="https://pkg.go.dev/text/template?utm_source=godoc"
+          target="__blank"
```

Comment:
I think when using `__blank` target anchors we usually add an icon to indicate this (`external-link-alt`)

Reply:
This link is inside the text. To use icons I would need to change it to be a button.